CONTRIBUTING.md

```diff
@@ -281,11 +282,21 @@ If you want to run a spell check of all `.Rmd` files locally, you can use run `R
 
 ## Rendering notebooks
 
+### Automatic rendering using GitHub actions
+
 This repository uses [snakemake](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html) to render all notebooks.
+All notebooks are automatically re-rendered by GitHub actions before merges to master.
+The newly rendered html files are all pushed to the `gh-pages` branch which will publish the material to https://alexslemonade.github.io/refinebio-examples/.
+
+If this automatic rendering fails, you can see the errors on GitHub by going to `Actions` and clicking the workflow of PR you are working on that also says `Build Docker` underneath.
+Click on `build` on the left bar and click on the step that has failed to see the error message.
+
+### Mechanics of the rendering
+
 The `Snakefile` calls the `scripts/render-notebooks.R` which renders the `.html` files but leaves these `.Rmd` files ready for download and use [without the `pandoc` error](https://github.com/AlexsLemonade/refinebio-examples/pull/148#issuecomment-669170681).
-`snakemake` should be ran after changes have been made and before any `Pull Request` are filed.
+However, `snakemake` can (and should) be run locally during development so that the author and reviewers can see the rendered output of the new material during the `Pull Request` process.
```

Comment:
I think we should keep the emphasis on "should"
```suggestion
However, the `snakemake` workflow should also be run locally during development so that the author and reviewers can see the rendered output of the new material during the `Pull Request` process.
```